Library/Formula/gitup.rb

```diff
@@ -0,0 +1,13 @@
+class Gitup < Formula
+  homepage "https://github.com/earwig/git-repo-updater"
+  url "https://github.com/earwig/git-repo-updater.git",
+    :revision => "51a482d7a25bed39ccc75780813c65bd666c1436"
```

Comment:
Please add the `:tag` as well as the `:revision`, Thanks! The addition of `tag` should make the `version` line redundant as well.